src/CaptivePortal.tsx

```diff
@@ -310,7 +310,7 @@ function CaptivePortalInner({
             <Button text="Hidden network..." />
             <HiddenNetwork onValidate={(essid, password) => connect(essid, password)} />
           </Popover>
-          <Button text="Use access point" onClick={startAp} />
+          <Button text="Use hotspot" onClick={startHotspot} />
```

Comment:
Only the text visible to the user was necessary to change.

You CAN change the name of the function that is triggered with the event but it was not requested.